scripts/lay-down-os

```diff
@@ -26,11 +25,6 @@ device_defined()
     fi
 }
 
-format_device()
```

Comment:
I moved this to set-disk-partitions.  Seems like the only time we should format is when the user uses the `-p` flag.  Other situations we just assume they formated.  This should allow somebody to use xfs or another file system if they choose.  RancherOS on boot essentially does `mount -t auto LABEL=RANCHER_STATE` so in theory other file systems *should* work.